apps/meteor/client/views/admin/apps/AppMenu.js

```diff
@@ -213,14 +233,27 @@ function AppMenu({ app, ...props }) {
 		showAppPermissionsReviewModal();
 	}, [action, app.id, app.purchaseType, cancelAction, checkUserLoggedIn, isAppPurchased, setModal, showAppPermissionsReviewModal]);
 
+	const incompatibleIconName = (app, action) => {
+		if (app.versionIncompatible) {
+			return 'warning';
+		}
+
+		switch (action) {
```

Comment:
why switch case, instead of if? there are only 2 options...

Reply:
That's true 🤔 I've removed it in favor of an early return. I've also removed the second parameter as it seems I wasn't using it so much.

Thanks 👊